xgcm/test/test_grid_ufunc.py

```diff
@@ -237,6 +237,63 @@ def diff_center_to_left(a):
         result = diff_center_to_left(grid, da, axis=[("depth",)])
         assert_equal(result, expected)
 
+    def test_1d_unchanging_size_but_padded_dask_parallelized(self):
+        """
+        This test checks that the process of padding a non-chunked core dimension doesn't turn it into a chunked core
+        dimension. See GH #430.
+        """
```

Comment:
I now have several tests which are very similar just with different dask chunking schemes, so I should try to refactored them as one parametrized test.